### PR TITLE
Org page

### LINKED ISSUE #779: Org page
We need an Org page, just like our project page. It shows:
- [ ] Name
- [ ] Members

And it's the workspace for org admins to:
- [ ] add members
- [ ] set member roles.

Permissions:
- Org admins
  - member CRUD (for now they will be auto added, but we may want that to be an invite process in the future)
- Org members
  - view member list
- Non org members
  - view org
  - list admins (including email tbd?), not other members

I assume we'll want the same member workflows we currently use for adding members to projects and setting their roles.

I think it would be very easy and reasonable to share a component with the project page for the general layout:
- Header
- Badges (Not sure what might go here for orgs)
- Attributes (description)
- History (eventually)
- More settings (delete, leave etc.)

Some minimal, but distinct style/color differences would be wonderful, so that the project page and org page can't be confused. And maybe so that the org page feels a tad "fancy", fresh and desirable.

Adding a project list to the org page is tracked in #788

### PR DESCRIPTION
**Pull request by rmunn**
Fixes #779.

Add an org page, parallel to the project page (and reusing several of its components), but with a slightly different layout, using tabs for the main body of the page.

Work done:

* Extracted common code from project page for header, details list (created date, description, etc.)
* Extracted members list code into common component
    * Currently hard-coded to use ProjectRole
* Edited MembersList component to use a generic role type that can be filled by either ProjectRole or OrgRole
* Used MembersList component in org page
* MembersList component now has a filter bar thanks to @myieye
* MembersList component now can switch between badge list and table view
* Separated MembersList component into project- and org-specific versions; org-specific one is now called OrgMemberTable
* MembersList component is back to using just ProjectRole, simplifying its code again
* Add "Add / Invite Member" button to table view so org page can use it

Deferred to other issues/PRs:

* Add "Bulk add / invite" button to table view so org page can use it
* Implement projects tab on org page (needs backend work so orgs can own projects)
* Implement settings tab on org page (will be separate issue)
* Implement history view on org page, or remove history tab if we're not implementing it anytime soon

Screenshot:

image

Previous screenshots:
1. [Click to view](https://github.com/sillsdev/languageforge-lexbox/assets/90762/ab51a201-a421-45de-a5f5-0704989cb6c1)
1. [Click to view](https://github.com/sillsdev/languageforge-lexbox/assets/90762/e9cb64ca-36f5-4619-96b7-b390e1784ae8)
1. [Members list view](https://github.com/sillsdev/languageforge-lexbox/assets/90762/fec6964c-2d03-4331-8e77-7c3a6246aa08)
1. [Members table view](https://github.com/sillsdev/languageforge-lexbox/assets/90762/d139d298-5366-415e-b0eb-0ea6d65c8e06)

### CODE REVIEW

**Comment by rmunn** [comment]
@hahn-kev @myieye - I'd like feedback on the members list component I just extracted. See PR description for screenshot; currently it looks and functions just like the members list for projects, except that I haven't yet implemented the "Add / Invite Member" button. (Or the bulk-add button). My concern, though, is as follows.

Some orgs might be small ("Language Name Here dictionary team") while others might be large (SIL). For large orgs, I think we'd want a member list layout more similar to the users tab on the admin page, but the MembersList component is quite useful for small orgs. Perhaps we automatically switch between them based on member count, or perhaps we provide a UI toggle for org admins to choose which view we want.

**Comment by rmunn** [comment]
@myieye - Commit 3d93f26488491774acc23c0310e8fd505e13389c reintroduces using roletext.toLowerCase() in the popup notification, but the role name for orgs is currently "user" and I had set the translated text to "member". Now the notification doesn't match what's in the role select choices. Should I change the translated role name to "User" rather than "Member" for orgs?

**Comment by rmunn** [comment]
Org page now has a users table where you can add and remove members. Currently adding is done with a button in the table header, and removing is done via a dropdown:

image

It feels a little cramped: I think there needs to be a bit of vertical space between the tabs and the table, and maybe some space above the tabs. But this is shaping up to be a functional, if still very basic, page. It might be time to call this done and move to the next task: allowing admins to create orgs, and/or allowing orgs to own projects.

**Comment by rmunn** [comment]
Lots of merge conflicts since I've been on this PR for a while, and the `develop` has gotten some major work merged recently. But the design work is finished. What's needed next is a bit of polishing of the UI (a bit more vertical space between the org details and the tab view, for example) and implementing any last-minute suggestions. But I'd say this is ready for a review from both the code and UI design perspectives.

**Comment by rmunn** [comment]
> @rmunn can you merge develop into this branch before we review is since it's so far behind?

Actually that's a good idea. When I run `git mergetool` and it autoresolves conflicts, it's *usually* correct, but I've sometimes found that manual correction of its resolution is needed. So by merging develop in before review, there will be more eyes looking at it in case I miss some merge-resolution mistake. Working on that now.

**Comment by rmunn** [comment]
My first attempt to merge `develop` into this branch went quite badly wrong as Git got confused by an earlier merge-and-revert I had done, and ended up choosing wrong branch resolutions all over the place. I've been able to rebase the branch to get rid of the merge-and-revert, and now Git is far less confused — and has only a single merge conflict, in the admin dashboard page. (I moved the users table to a separate component, while meanwhile Tim made a two-line change in the users table on the admin dashboard).

I should be able to resolve that merge conflict by hand, without having to merge `develop` into this branch.

**Comment by rmunn** [comment]
Rebased on top of develop instead of merging, and everything went much more smoothly. No merge conflicts any more.

@myieye @hahn-kev Ready for review now: merge conflicts are all solved.

File: frontend/src/lib/layout/DetailItem.svelte
Comment (hahn-kev) [inline frontend/src/lib/layout/DetailItem.svelte:7]: not sure how I feel about making this an i18n key, while placeholder is a string and the translation is pushed to the user of this component. I think I'd rather this be the translated text already to stay consistent with our other components.

File: frontend/src/lib/components/MembersList.svelte
Comment (hahn-kev) [inline frontend/src/lib/components/MembersList.svelte:4]: I think this property can be required and not null.

File: frontend/src/routes/(authenticated)/project/[project_code]/ChangeMemberRoleModal.svelte
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/project/[project_code]/ChangeMemberRoleModal.svelte:12]: why is this now a $ property?

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (hahn-kev) [inline frontend/src/lib/components/Orgs/OrgMemberTable.svelte:11]: it feels like we should't be importing a component in a route from lib. Should we either move the org table component into the route or the add org member component into lib?

File: frontend/src/lib/forms/OrgRoleSelect.svelte
Comment (hahn-kev) [inline frontend/src/lib/forms/OrgRoleSelect.svelte:15]: disabled is not passed in here

File: frontend/src/lib/forms/OrgRoleSelect.svelte
Comment (hahn-kev) [inline frontend/src/lib/forms/OrgRoleSelect.svelte:24]: on change is only supported when not using the label, not sure that makes much sense. Can we make it consistent?

File: frontend/src/lib/layout/DetailItem.svelte
Comment (hahn-kev) [inline frontend/src/lib/layout/DetailItem.svelte:12]: not sure I love this. This component does 2 things, it's either an editor or a display, will we ever need to switch between the 2?

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:38]: if we don't need the warning then just remove it instead of commenting it out

**Review by hahn-kev — CHANGES_REQUESTED** [review]
nice work Robin, I'm glad we have an org page to actually view and show org members. I think there's some cleanup that still needs to be done, but this PR has gone on for a while now. For the simple stuff just fix it here, otherwise make a new issue as I'm fine to defer the larger tasks.

**Review by myieye — CHANGES_REQUESTED** [review]
I think this PR is going in a good direction. I think you're aware that there's still stuff to do:
- Actions/buttons to implement
- Commented out code to remove
- Etc.

I addressed most of @hahn-kev's comments except for the ones regarding missing functionality. I also pushed a number of other changes:
- I removed the role dropdowns from the table, because they don't work well with SSR and I think they're more noisy than they're worth. It also simplified the code a fair bit
- I moved the date values into History, because I think they're less valuable for orgs than for a project. I don't think most users would really be interested in them.
- I removed the badges, because they're redundant: we have that information in the tab headers and I like it there more
- I made the Add member button just an Icon, because I think it should be intuitive enough and it's actually a dropdown 🤷 
image
- I put some buttons in "Settings", but didn't implement them
- I made the badge code in the tabs less generic and simpler

**Comment by myieye** [comment]
@rmunn FYI, if you dislike any of my layout/structure changes feel free to revert them.
I was just trying to move fast and there are so many little decisions to make that I didn't think waiting was a good idea.

**Comment by rmunn** [comment]
Here's why I made the OrgRoleSelect able to remove the label: I wanted to be able to use it in both forms and in the table, and in the table, the label gets in the way rather badly. Here's a GIF of using the select in the table with the label present:

select-with-label

And here's a GIF of using it without the label:

select-without-label

The table still pops around because of the different size of the select without the label, but I can fix that by giving the table cell an appropriate size.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (myieye) [inline frontend/src/lib/components/Orgs/OrgMemberTable.svelte:28]: Please make the dispatch type-save.
Your `removeMember` handler is actually broken right now.
Type-safety protects us from that sort of thing.

File: frontend/src/lib/forms/OrgRoleSelect.svelte
Comment (myieye) [inline frontend/src/lib/forms/OrgRoleSelect.svelte:10]: This no longer has a label in the add member modal.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (myieye) [inline frontend/src/lib/components/Orgs/OrgMemberTable.svelte:78]: This is not implemented yet

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (myieye) [inline frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:55]: This is pretty hard core 🤔
If you really think it's a good idea, I think it's critical that you translate these texts.

**Review by myieye — CHANGES_REQUESTED** [review]
Good work Robin 😃, but not quite there yet 😬 

#### 1) The select in the table
Ah, I see. I was sure that you were going to open a modal from the three-dot-menu for changing the role.

There are 2 reasons I don't love the select-list in the table:
- It's a new UI pattern for us
- There's no submit or confirmation step after selecting the new role

A modal is the easiest way to fix those. Another option would be something similar to our in-place org name editor. But, making a select version of that is somewhat non-trivial and it almost always gets a bit awkward trying to fit 2 different elements (the editor and the value) in one place.

#### 2) I've found a few broken/missing implementations

#### 3) Regarding commented out code and open TODOs

I'm trying to figure out what a good compromise is 🤔, because you have quite a unique approach in this area. There's so much commented out code in here that you copy-pasted, but also updated to make it org-specific. As a result there are lots of translations that aren't being used, there are TODOs that I don't think anyone will explicitly search for, but rather remove as they come accross them.

Doesn't it generally make sense to only copy-paste and adapt code when we actually need it instead of before hand? There's the risk that we'll have dead code for a long time and by checking it in, it's looks like it got reviewed, but it was never tested and I'm sure not looking at it closely. Then there's the risk that when it gets uncommented we'll treat it as reviewed code instead of new code.

I'd suggest removing the code and todos for:
- search-params
- org descriptions

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (hahn-kev) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:55]: Yeah I think this is overkill as long as we prompt the user to type something in (like DELETE_ORG which we do for projects and such). I also think we can implement a soft delete later on. I'll make an issue for that.

Comment (myieye) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:55]: Yeah, I do like the idea of a soft delete.

**Comment by myieye** [comment]
@rmunn I just pushed a commit, that I meant to push Yesterday: I tried a new sport for the Add/Invite member button, namely in the page actions. It seems ok to me to not put it directly by the members and have it always visible. I wasn't quite happy with where I had it before.

I'm not quite sure where the bulk-add button would go, but there are options.

image

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte]: Done in commit c6c6d7ec. Thanks for the tip; I'd forgotten that createEventDispatcher can be made type-safe. (I'd seen it elsewhere in our code but forgot about it while I was deep in the org page code).

File: frontend/src/lib/forms/OrgRoleSelect.svelte
Comment (rmunn) [reply frontend/src/lib/forms/OrgRoleSelect.svelte:10]: Yep, which is why I had the `{#if withLabel}` param earlier which disappeared in one of the refactors. Since we're going to be moving it into the edit-user modal and NOT sticking the select into the table, I'll go ahead and put the label back; the whole reason for its removal was so that the select-in-the-table thing would look good (c.f. gifs above).

I still like the select-in-the-table idea, and I might try to bring it back later on with an edit icon (pencil) that appears on hover and turns the text box into a role select. BUT... now's not the time to get fancy with this, so I'll put that off until later.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte:56]: Yep. Working on it alongside the editUser modal that the select-in-the-table is going to turn into. Won't get it finished today, but wanted to acknowledge your comment so you know I've seen it and will work on it.

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:53]: Once we move to using an `isDeleted` column on orgs the way we do on projects, the double-confirm won't be necessary. Maybe I'll change it to only trigger if the change would affect more than 10 projects or more than 25 users, that way deleting test orgs that we create during development won't be tedious. And eventually it can go away.

But yes, certainly, it needs to be translated, which is why I put a TODO in there to remind myself of that. (Also why I'm increasingly feeling like it's long past time to put https://github.com/sillsdev/languageforge-lexbox/issues/602 into the next milestone we do, to make sure we don't populate en.json and neglect the translations).

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (myieye) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte:58]: Don't forget: we already have a modal for displaying the user's details! 🙂

File: frontend/src/lib/forms/OrgRoleSelect.svelte
Comment (rmunn) [reply frontend/src/lib/forms/OrgRoleSelect.svelte]: Fixed in commit a0b3bf1b where I move to using the modal, rather than select-in-table, to more closely parallel the project page.

**Comment by rmunn** [comment]
@myieye - Many comments addressed, but ran out of time to do all of them. Will finish Monday.

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte]: Changed my mind and decided that our existing confirm dialog is good enough; by the time we have orgs with many projects in them, we'll likely have a soft-delete option. So I removed this in commit 909070b5.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte:58]: I didn't forget; as you'll see in commit 9f2e0d6c, I used the same modal.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte]: Implemented now, in commit 9f2e0d6c.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte]: Changed to using an edit-user modal same way the project page does; it now actually makes changes.

File: frontend/src/lib/components/Orgs/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/lib/components/Orgs/OrgMemberTable.svelte]: Now it calls up the edit-user modal (and I've changed the event to be called `changeMemberRole`).

**Comment by rmunn** [comment]
I started implementing the "invite org member by email" flow and realized that it's going to take a bit more effort than I thought it would: we can't simply reuse the existing invite project member by email without changes. We'll need to change the acceptInvitation page to look for a list of orgs to join as well as projects to join, we'll need to edit the invitation JWT to optionally include a list of orgs as well as a list of projects...

I think it's best if I push that work off to a separate PR, so that the org page work can be completed. So for now, I'll leave the "TODO: Implement inviting user" comment in OrgMutations.cs, as a reminder for us to do that work. (I'll also create an issue for it).

**Comment by rmunn** [comment]
I looked into Scribe, and it requires either a Chrome or Edge extension. I uninstalled Chrome from my Linux computer ages ago because it's getting increasingly privacy-hostile (I use Chromium if I need to test something in Chrome), so I really don't want to reinstall it just for this. @myieye, could I ask you to create an Add_Org_Member scribe similar to the Add_Project_Member one?

File: frontend/src/routes/(authenticated)/project/[project_code]/ChangeMemberRoleModal.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/project/[project_code]/ChangeMemberRoleModal.svelte]: Leftover from when the component handled both project and org memberships and needed to change its schema based on the `roleType` prop. Removed in commit 739524f0.

**Comment by rmunn** [comment]
Only problem remaining is the Typescript errors that result from differing definitions of User/TableUser/etc. I don't have a good idea for how to fix them (my earlier attempts have led to much confusion), but perhaps I'm simply missing something obvious. @myieye, could I ask you to take a look and see if you spot something I missed? Run `cd frontend; pnpm check` and you'll get several errors along the lines of "Argument of type 'Pick<User, "id" | "name" | "email" | "username" | "isAdmin" | "emailVerified" | "locked">' is not assignable to parameter of type 'User'".

**Comment by rmunn** [comment]
It looks to me like the TableUser type in [`frontend/src/lib/components/Users/UserTable.svelte`](https://github.com/sillsdev/languageforge-lexbox/blob/c6c6d7ec064161084a3d0fba464f448738a8ed0d/frontend/src/lib/components/Users/UserTable.svelte) is at the center of the Typescript issues I'm running into. If I simply define it as User then some of the errors go away, but I still get things like:

```
/home/rmunn/code/lexbox/frontend/src/routes/(authenticated)/admin/+page.svelte:159:12
Error: Type '{ id: string; name: string; email?: string | null | undefined; username?: string | null | undefined; isAdmin: boolean; emailVerified: boolean; createdDate: string | Date; locked: boolean; ... 4 more ...; projects: { ...; }[]; }[]' is not assignable to type 'User[]'.
  Type '{ id: string; name: string; email?: string | null | undefined; username?: string | null | undefined; isAdmin: boolean; emailVerified: boolean; createdDate: string | Date; locked: boolean; ... 4 more ...; projects: { ...; }[]; }' is missing the following properties from type 'User': organizations, usersICreated (ts)
```

I'm too close to the problem to be able to see it clearly any more. I think I need input from someone who hasn't been staring at this code for a month. :-)

**Review by myieye — APPROVED** [review]
@rmunn I just made a decently sized push:
- [461b789](https://github.com/sillsdev/languageforge-lexbox/pull/800/commits/461b789d3f2f838634bb8844cf9cf5639ef5e777) Fixed the TypeScript errors
- [11cf0b9](https://github.com/sillsdev/languageforge-lexbox/pull/800/commits/11cf0b9a94a9125ef4ab2182082bf00ca3218cd2) maybe shouldn't be in this PR, but I was having lots of bizarre problems with SK and updating fixed them 🙃 
- [4faa6df](https://github.com/sillsdev/languageforge-lexbox/pull/800/commits/4faa6dfea32a6fb921f6a97da456e8d00a29db17) is the big one. I did several things:
  - I realized that some components were still generalized for handling org or project roles or verbose vs non-verbose texts, but that the generalisations weren't needed anymore
  - I moved a handfull of components back to the page that their page.ts/.svelte is in, because it makes them easier to find and in some cases there were still imports going back to the page files, so this cleaned them up
  - I've adaopted the practice of exporting some types from page.ts that the svelte page and components can share (e.g. a page-specific User type or OrgUser type etc.)
  - I made your Org role display name component work the same way as the project role display name component
  - I standardized some texts on the org page to be more similar to the project page

**Comment by rmunn** [comment]
> @rmunn I just made a decently sized push:
> 
> * [461b789](https://github.com/sillsdev/languageforge-lexbox/pull/800/commits/461b789d3f2f838634bb8844cf9cf5639ef5e777) Fixed the TypeScript errors

:tada: I knew there would be an easy solution that I was just too close to the code to see. Thanks!

I've done a self-review and found one bug, so I pushed commit c9240d6e to fix it. Everything else looks good to me. I think this is finally ready.

@hahn-kev, you have the last remaining "Changes requested" review. Do you want to give this another look?

File: frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte:60]: this shows 'undefined' when you view the member table as a non admin
firefox_W4wtZTNYSd. We should probably just hide the column for non admins.

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: this button should only show if you're a member of the org. If I login as a user who's not in this org and access the org page (which is allowed) then I can't actually leave the org if I'm not a member. Maybe we should just hide the settings tab if the user isn't a member and not an admin.

**Review by hahn-kev — CHANGES_REQUESTED** [review]
a couple minor things, then this can be merged in. I also noticed a bit of a permissions issue so I created #879 to track that.

File: frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte:60]: Sure. At some point we may decide to expose email addresses to other org members, at which point we may bring the header back. But for now, hiding that column for non-admins (since non-admins don't get to see the information anyway) makes sense.

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: Yeah, a non-member shouldn't even see the Settings tab if there's nothing they can do there. No point in wasting screen real estate.

File: frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte
Comment (myieye) [reply frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte:60]: Oh right, non-admins 🤦. Non-admins also shouldn't:
- See the member actions menu
- Be able to click on a user to view their details

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (myieye) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: A super-admin should see the tab and be able to delete the org.

Wait, what? I don't think you should be allowed to access the org page or query an org that you're not a member of.
Then everyone would be able to see everyone's names and projects. Super-admins, don't have to be members of course.
But I've only just realized that none of our org GQL queries have any permissions checks on them. 

1) I think you should only be able to query orgById if you're a member of the org.
2) And we need to lock down what fields non-members can query, so that they can see the list of orgs. We could do that by returning a DtoOrg with limited fields or...

We could accomplish both requirements at once, by doing something like: `descriptor.Field(org => org.Members).OrgMembershipRequired();`

Comment (megahirt) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: good thinking ya'll !  Do you think these things should hold up the PR or just be deferred to a new issue?  I'm afraid of this PR sliding into its fifth week...

**Comment by myieye** [comment]
I propose you merge this PR right now and finish the TODOs in the scope of this new issue I created:
https://github.com/sillsdev/languageforge-lexbox/issues/881

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (myieye) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: [Beat you](https://github.com/sillsdev/languageforge-lexbox/pull/800#issuecomment-2164820236) 😆 👏

File: frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte
Comment (myieye) [reply frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte:60]: See https://github.com/sillsdev/languageforge-lexbox/pull/800#issuecomment-2164820236

**Comment by rmunn** [comment]
> I propose you merge this PR right now and finish the TODOs in the scope of this new issue I created: #881

I'd already implemented the settings tag change before you said that, but I hadn't committed it yet. I was thinking of pushing that commit then asking for thoughts, but instead I'll go ahead and make that commit part of the new branch for #881.

I'll dismiss Kevin's review since you've created the issue to track those remaining tasks, and merge this.

File: frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte:135]: Will do this as part of #881.

File: frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte
Comment (rmunn) [reply frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte:60]: Will do this as part of #881.

**Pull request merged**
